src/commcare_cloud/ansible/roles/devops_scripts/templates/autoreboot.sh.j2

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+current_availability_zone=$(ec2metadata --availability-zone | awk -F'-' '{print $3}')
```

Comment:
This would be more general if you pull of just `a` instead of `1a`, because the `1` is part of the region

Reply:
modifying the command as suggested above **ec2metadata --availability-zone | awk -F'-' '{print $3}' | cut -c2-** 
it uses cut to remove the character from the output **-c, --characters=LIST   select only these characters**. Inour case it pulling of just a from 1a.